src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -17,6 +17,8 @@
 
 #include "DataStructures/DataBox/DataBoxTag.hpp"
 #include "DataStructures/DataBox/Deferred.hpp"
+#include "DataStructures/DataBox/Item.hpp"
```

Comment:
Some typos in commit message of ` Replace DataBoxLeaf with Item in implementation of DataBox `

- second line: `of a set` -> `from a set`?